modules/UI/util/UIUtil.js

```diff
@@ -103,13 +103,17 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             'top-right': 'sw'
         };
 
-        element.setAttribute("data-i18n", "[content]" + key);
+        $(element).each(function () {
+            this.setAttribute("data-i18n", "[content]" + key);
+        });
         APP.translation.translateElement($(element));
 
         AJS.$(element).tooltip({
             gravity: positions[position],
             title: this._getTooltipText.bind(this, element),
-            html: true
+            html: true,
+            live: true,
```

Comment:
I couldn't find any documentation for `live` and had to look at the implementation to get a (vague) idea. Could you please provide a source code comment on `live` explaining why it's necessary, the problem that it fixes, etc.?